### DISCUSSION TITLE
Usage with dom created by pure javascript.

### DISCUSSION THREAD

**Discussion by Saul-Mirone** [unocss/unocss#255] [Q&A]
Thank you for creating this awesome project. I'm considering using this project. However, in my case, there's a lot of dom created and managed by javascript, like this:

```javascript
const dom = document.createElement('div');
dom.appendChild(otherDom);
```

Can `unocss` be used in this scenario?

**Comment by antfu** [comment]
By default UnoCSS does not scan js/ts files. https://github.com/antfu/unocss#scanning

You can enable them by

```ts
Unocss({
  includes: [/\.[jt]sx?$/]
})
```

https://github.com/antfu/unocss/blob/c7007695b198c4838276db2a652abdf3fc63cdea/packages/plugins-common/defaults.ts#L1-L3
